README.md

```diff
@@ -1,14 +1,11 @@
 # Hypha Co-op: Shortlinks
 
-For generating our `link.hypha.coop` shortlinks.
+Source for our `link.hypha.coop` shortlinks
 
 Configuration for these shortlinks are stored in
 [`shortlinks.csv`](/shortlinks.csv).
 
-This repo contains only configuration and documentation. All the
-updating happens through the `update_shortlinks` task of the
-[`hyphacoop/worker-coop-scripts` repo][scripts]. Please see that repo
-for details.
+This repo contains a CSV file used by the [hyphacoop/shortlinks-site] script and this documentation.
```

Comment:
Where is the link?

Reply:
Bad github shortcut linking. Corrected with full URL